### ISSUE TITLE
Macrophage Activation Syndrome

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6063]
**Preferred term label:**
Macrophage Activation Syndrome

**Synonyms**
Haemophagocytic lymphohistiocytosis, Reactive haemophagocytic syndrome, Hemophagocytic syndrome, MAS

**Definition (free text, please give PubMed ID)**
Uncontrolled activation and proliferation of T lymphocytes and macrophages
PMID: 28095869

**Parent term (use hpo.jax.org/app)**
Immune dysregulation

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: #603553

**Gene association**
PRF1 (OMIM: 170280)

**Comment by pnrobinson** [comment]
This is a disease, we would use this Mondo term: https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0015545&viewMode=All&siblings=false
